pkg/gits/git_cli.go

```diff
@@ -506,7 +506,7 @@ func (g *GitCLI) CommitDir(dir string, message string) error {
 }
 
 // AddCommit perform an add and commit of the changes from the repository at the given directory with the given messages
-func (g *GitCLI) AddCommit(dir string, msg string) error {
+func (g *GitCLI) CommitAllChangesToTrackedFiles(dir string, msg string) error {
```

Comment:
The godoc is out of sync here.

That said, why do you want to rename the function? Is `CommitAllChangesToTrackedFiles` not a bit superfluous.  `AddCommit` (or even better `Commit`)  does already imply that, no?

Personally, the new name confuses me more instead of clarifying anything, since I am wondering at that stage what the semantic difference to a 'commit' would be.

Reply:
@hferentschik To me, `AddCommit` is ambiguous because it could mean that tracked & untracked files are added, or it could mean simply `git commit -a` which only adds tracked files, the latter of which is the behavior of `AddCommit`. I admit that it's a wordy name! But it tells you exactly what the method is doing.

Reply:
Ok. Maybe `CommitChangedTrackedFiles` is a bit friendlier?